src/DripsDeployer.sol

```diff
@@ -319,94 +315,51 @@ contract RepoDriverModule is CallerDependentModule, DriverModule(3) {
     }
 }
 
-/// @notice Creates a contract under a deterministic addresses
-/// derived only from the deployer's address and the salt.
-/// The deployment is a two-step process, first, a proxy is deployed using CREATE2 with
-/// the given salt, and then it's called with the bytecode of the deployed contract,
-/// the proxy uses it to deploy the contract using regular CREATE.
-/// If the deployed contract's constructor reverts, the proxy also reverts.
-/// If the proxy call has a non-zero value, it's passed to the deployed contract's constructor.
-/// Based on the bytecode from https://github.com/0xsequence/create3.
-library Create3 {
-    using Address for address;
-
-    //////////////////////////// PROXY CREATION CODE ///////////////////////////
-    // Opcode     | Opcode name      | Stack values after executing
-    // Store the proxy bytecode in memory
-    // 0x67XX..XX | PUSH8 bytecode   | bytecode
-    // 0x3d       | RETURNDATASIZE   | 0 bytecode
-    // 0x52       | MSTORE           |
-    // Return the proxy bytecode
-    // 0x6008     | PUSH1 8          | 8
-    // 0x6018     | PUSH1 24         | 24 8
-    // 0xf3       | RETURN           |
-
-    ////////////////////////////// PROXY BYTECODE //////////////////////////////
-    // Opcode     | Opcode name      | Stack values after executing
-    // Copy the calldata to memory
-    // 0x36       | CALLDATASIZE     | size
-    // 0x3d       | RETURNDATASIZE   | 0 size
-    // 0x3d       | RETURNDATASIZE   | 0 0 size
-    // 0x37       | CALLDATACOPY     |
-    // Create the contract
-    // 0x36       | CALLDATASIZE     | size
-    // 0x3d       | RETURNDATASIZE   | 0 size
-    // 0x34       | CALLVALUE        | value 0 size
-    // 0xf0       | CREATE           | newContract
-
-    bytes private constant PROXY_CREATION_CODE =
-    // Proxy creation code up to `PUSH8`
-        hex"67"
-        // Proxy bytecode
-        hex"363d3d37363d34f0"
-        // Proxy creation code after `PUSH8`
-        hex"3d5260086018f3";
-
-    /// @notice Deploys a contract under a deterministic address.
+/// @notice Deploys contracts using CREATE3.
+/// Each deployer has its own namespace for deployed addresses.
+library Create3Factory {
+    /// @notice The CREATE3 factory address.
+    /// It's always the same, see `deploy_create3_factory` in the deployment script.
+    ICreate3Factory private constant _CREATE3_FACTORY =
+        ICreate3Factory(0x6aA3D87e99286946161dCA02B97C5806fC5eD46F);
```

Comment:
Okay, so this wrapper library is a trick that you only need to hardcode the address  once, correct.